packages/design-system-v2/src/box.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
What's this?

Reply:
This I need to test storybook hosting. Will remove afterwards